starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +21,51 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord (array) {
+  var longestWord="";
+  for (var i = 0; i<array.length; i++) {
+    if (array[i].length>longestWord.length) {
+      longestWord=array[i]
+    } 
+  }
+  return longestWord
+}
+console.log(findLongestWord(words))
+
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumArray(array) {
+  var sum = 0;
+  for (var i= 0; i<array.length; i++) {
+    sum +=array[i]
+  }
+  return sum
+}
+console.log(sumArray(numbers))
+
+// Sum using reduce
+var sum = numbers.reduce(function(accumulator, currentValue){
+  return accumulator + currentValue
+},0)
+console.log(sum)
+
```

Comment:
Very nice Ana that you implemented the reduce, your getting the hang of it.
Let's level up from now on and use ECMA 6 with let and const instead of var and the use of =>
Also one of the main features of higher functions like reduce is that it has less code.
If we combine the reduce method above with ECMA 6, we get even less, but more clean code:
`const sum = numbers.reduce((accumulator, currentValue) => {
return accumulator + currentValue;
}, 0);`